test/language/directive-prologue/10.1.1-2gs.js

```diff
@@ -9,10 +9,10 @@ es5id: 10.1.1-2gs
 description: >
     Strict Mode - Use Strict Directive Prologue is ''use strict''
     which lost the last character ';'
-negative: NotEarlyError
-flags: [noStrict]
```

Comment:
Can we keep `noStrict` here so that we (V8) do not run this test in both strict and sloppy mode?